README.md

```diff
@@ -52,11 +54,18 @@ An **opaque-blocklisted-never-sniffed MIME type** is a MIME type whose essence i
 * "`text/event-stream`"
 * "`text/csv`"
 
-A user agent has an **opaque-safelisted requesters set**. (This should be scoped similar to other network caches.)
+### Changes to requests and media elements
+
+A request has an associated **no-cors media URL** ("N/A", "initial-request", or a URL). It is "N/A" unless explicitly stated otherwise.
```

Comment:
This could be a tri-state as all we really need to know is that it's the subsequent request. Would that be better? Or would that even further hide the complexity for implementations that want to protect against a compromised process?

Reply:
Tri-state seems slightly better because:

- Having a name for non-initial media request seems helpful.  Not sure about the exact name: "non-initial-media-request"?  "repeated-media-request" (my vote)?  "subsequent-media-request"?

- This allows for a succint description of how a trustworthy process could implement this (I'll propose something below).

- This allows simplification of

> If _request_'s no-cors media URL is a URL and it is equal to _request_'s current URL, then return true.

with a simpler:

> If _request_'s no-cors media request state is "repeated-media-request" , then return true.